specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/Jobs.json

```diff
@@ -352,8 +352,8 @@
           {
             "name": "template",
             "in": "body",
-            "description": "Properties used to start a job instance.",
-            "required": true,
+            "description": "Properties used to start a job execution.",
+            "required": false,
```

Comment:
This is breaking change and needs approval.

Reply:
@raych1 since this merge is against is a feature branch, we see this difference. Though the change to the `required` parameter has already been completed in these versions in the main branch:
https://github.com/Azure/azure-rest-api-specs/blob/23d88533ddfde4d1565a897fe95d42fb0d9333e5/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/Jobs.json#L356